src/renderer/marketplace/bank-accounts/list/offers-table.jsx

```diff
@@ -133,24 +134,25 @@ export const BankingOffersTable = withStyles(styles)(
 					{data.map(bank => (
 						<TableRow key={bank.id}>
 							<TableCell className={classes.flagCell}>
-								<FlagCountryName code={bank.countryCode} size="small" />
+								<FlagCountryName code={bank['Country Code']} size="small" />
 							</TableCell>
-							<TableCell className={classes.regionCell}>{bank.region}</TableCell>
+							<TableCell className={classes.regionCell}>{bank['Region']}</TableCell>
 							<TableCell className={classes.eligibilityCellBody}>
-								{bank.eligibility}
+								{bank['Eligibility'] &&
+									bank['Eligibility'].map(tag => <Tag key={tag}>{tag}</Tag>)}
 							</TableCell>
 							<TableCell className={classes.minDepositCell}>
-								{bank.minDeposit} {bank.minDepositCurrency}
+								{bank.minDeposit} {bank['Min Deposit']}
 							</TableCell>
 							<TagTableCell className={classes.goodForCell}>
-								{bank.goodFor &&
-									bank.goodFor.map(tag => <Tag key={tag}>{tag}</Tag>)}
+								{bank['Good For'] &&
+									bank['Good For'].map(tag => <Tag key={tag}>{tag}</Tag>)}
 							</TagTableCell>
 							<TableCell className={classes.personalVisitCell}>
 								{bank.personalVisit ? 'Required' : 'No'}
 							</TableCell>
 							<TableCell className={classes.costCell}>
-								<ProgramPrice label="$" price={bank.price} rate={keyRate} />
+								<ProgramPrice label="$" price={getBankPrice(bank)} rate={keyRate} />
```

Comment:
`getBankPrice` should not exist, it should come already reformatted via selector

Reply:
Agreed. This is now handled on the selector.